lib/SILOptimizer/Mandatory/DefiniteInitialization.cpp

```diff
@@ -783,6 +786,155 @@ void LifetimeChecker::diagnoseInitError(const DIMemoryUse &Use,
     diagnose(Module, TheMemory.getLoc(), diag::variable_defined_here, isLet);
 }
 
+static void injectHopToExecutorBefore(SILLocation loc,
+                                      SILBasicBlock::iterator &insertPt,
+                                      SILValue actor, bool needsBorrow) {
+  SILBuilder B(insertPt);
+
+  LLVM_DEBUG(llvm::dbgs() << "hop-injector: inserting hop_to_executor before ");
+  LLVM_DEBUG(insertPt->dump());
```

Comment:
Instead of using dump here, please just pass *insertPt to the line before. dump is meant to be just used in the debugger.

Reply:
Example: https://github.com/apple/swift/blob/191dc14c59a96b2c9a38540e3f3e8624e1e7404d/lib/SILOptimizer/Mandatory/DefiniteInitialization.cpp#L3177